cli/dts/lib.deno.ns.d.ts

```diff
@@ -2455,19 +2455,9 @@ declare namespace Deno {
    * Subprocess uses same working directory as parent process unless `opt.cwd`
    * is specified.
    *
-   * Environmental variables from parent process can be cleared using `opt.clearEnv`.
-   * Doesn't guarantee that only `opt.env` variables are present,
-   * as the OS may set environmental variables for processes.
-   *
    * Environmental variables for subprocess can be specified using `opt.env`
    * mapping.
    *
-   * `opt.uid` sets the child process’s user ID. This translates to a setuid call
-   * in the child process. Failure in the setuid call will cause the spawn to fail.
-   *
-   * `opt.gid` is similar to `opt.uid`, but sets the group ID of the child process.
-   * This has the same semantics as the uid field.
```

Comment:
Why remove these comments?

Reply:
These no longer exists in [`Deno.RunOptions`](https://doc.deno.land/deno/stable/~/Deno.RunOptions).

Reply:
They definitely exist, they are just unstable options

Reply:
Hmm, [you are right](https://doc.deno.land/deno/unstable/~/Deno.run), but shouldn't this be in `lib.deno.unstable.d.ts` like there is both [stable](https://doc.deno.land/deno/stable/~/Deno.connect) and [unstable](https://doc.deno.land/deno/unstable/~/Deno.connect) comments about `Deno.connect`?

Reply:
It should be noted that they require the unstable flag to work in the comment.